sdk/identity/Azure.Identity/src/Credentials/ChainedTokenCredential.cs

```diff
@@ -76,6 +76,8 @@ public ChainedTokenCredential(params TokenCredential[] sources)
         /// <param name="requestContext">The details of the authentication request.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
         /// <returns>The first <see cref="AccessToken"/> returned by the specified sources. Any credential which raises a <see cref="CredentialUnavailableException"/> will be skipped.</returns>
+        /// <exception cref="AuthenticationFailedException">Thrown when the authentication failed.</exception>
+        /// <exception cref="CredentialUnavailableException">Thrown when the credential is unavailable when used as part of a chained credential such as <see cref="DefaultAzureCredential"/> or <see cref="ChainedTokenCredential"/>.</exception>
```

Comment:
I'm not sure this wording makes sense for CTC. CTC itself would never be unavailable, but credential types within it may be.